simson/common/data_extrapolations.py

```diff
@@ -37,15 +37,24 @@ def predict(self):
 
 
 class ExponentialExtrapolation(Extrapolation):
-    initial_guess: np.ndarray = np.array([400, 1])  # these values work well for stock predictions
-    # in the steel model, but should be passed on initialisation for other usecases.
+
+    def initial_guess(self):
+        current_level = self.data_to_extrapolate[-1]
+        current_extrapolator = self.extrapolate_from[self.n_historic - 1]
+        initial_saturation_level = 2. * current_level if current_level != 0.0 else 1.0
+        initial_stretch_factor = - np.log(1 -  current_level / initial_saturation_level) / current_extrapolator
+
+        return np.array([initial_saturation_level, initial_stretch_factor])
 
     def fitting_function(self, prms):
         return (
             prms[0] * (1 - np.exp(-prms[1]*self.extrapolate_from[:self.n_historic]))
         ) - self.data_to_extrapolate
 
     def predict(self):
-        prms_out = least_squares(self.fitting_function, x0=self.initial_guess, gtol=1.e-12)
+        # Todo: Decide whether to use bounds
+        # bounds = ([0., 0.], [30, np.inf])
+        prms_out = least_squares(self.fitting_function, x0=self.initial_guess(), gtol=1.e-12) #, bounds=bounds)
```

Comment:
This option (bounds, no bounds) could be passed as a parameter to this predict method.

Reply:
I think that's a good idea - for now, to have some cleaner code, I'm getting rid of the bounds parameter whilst waiting for Jakob to reevaluate the extrapolation altogether (right now the values don't make that much sense anyway). But I think this could be a good option for the future depending on how adaptable we want the model to be.